neo/rawio/openephysbinaryrawio.py

```diff
@@ -174,12 +181,21 @@ def _parse_header(self):
         signal_buffers = []
 
         unique_streams_ids = np.unique(signal_channels["stream_id"])
+        
+        # This is getting too complicated, we probably should just have a table which would be easier to read
+        # And for users to understand
         for stream_id in unique_streams_ids:
-            # Handle special case of Synch channel having stream_id empty
-            if stream_id == "":
+            
+            # Handle sync channel on a special way
+            if "SYNC" in stream_id:
+                # This is a sync channel and should not be added to the signal streams
+                buffer_id = sync_stream_id_to_buffer_id[stream_id]
+                stream_name = stream_id
```

Comment:
As suggested here (https://github.com/SpikeInterface/spikeinterface/pull/3839#issuecomment-2785810190), we should keep track of the record node that the sync channel came from. Something like this?
```suggestion
                stream_name = sig_stream_names[stream_index].split('#')[0] + '#' + stream_id
```

Reply:
Ok, this works, but there could be a better way
```suggestion
                stream_index = int(stream_id.split('-')[-1])
                stream_name = sig_stream_names[stream_index].split('#')[0] + '#' + stream_id
```

Reply:
@chrishalcrow this would only keep the record node and not the OE stream name though, right?

Reply:
I changed this already, can you test it @chrishalcrow?

Reply:
Changed this to use `{stream_name}SYNC` for both id and name.

Reply:
Tested - all works for the "real world" cases I'm trying!